pkgs/tools/networking/v2ray/default.nix

```diff
@@ -50,8 +50,8 @@ let
     buildPhase = ''
       runHook preBuild
 
-      go build -o v2ray v2ray.com/core/main
-      go build -o v2ctl v2ray.com/core/infra/control/main
+      go build -v -p $NIX_BUILD_CORES -o v2ray ./main
```

Comment:
Can we add -ldflgas="-s -w"?

Reply:
@SuperSandro2000 
`-s` is for strip, but we already have default fixupPhase to strip all executables. 
What's `-w`? `man ld` doesn't contain it.

Reply:
Please add it anyway explicitly to ldflags.

https://stackoverflow.com/questions/22267189/what-does-the-w-flag-mean-when-passed-in-via-the-ldflags-option-to-the-go-comman

Reply:
Fixed.

Reply:
@SuperSandro2000 It do reduce the executable size. Thanks.